blueprints/batch-jobs/README.md

```diff
@@ -0,0 +1,214 @@
+This section simulates the default behaviour of the pods explained before, in which the Karpenter consolidation process interrupts the pods running the jobs, and re-schedule them into the cheaper node. To simulate it, deploy the [workloads-evicted yaml](/karpenter-blueprints/blueprints/batch-jobs/workloads-evicted.yaml):
+```
+$> kubectl apply -f workloads-evicted.yaml
+deployment.apps/nginx created
+job.batch/2-min-job created
+job.batch/5-min-job created
+```
+This will create three pods that require **11 vCPU** in total:
+-  NGINX server - 2 vCPU required
+-  2-minutes job - 7 vCPU required
+- 5-minutes job - 2 vCPU required
+
+Karpenter launches the cheapest EC2 instance for the workloads: a **c6g.4xlarge** on-demand instance (16 vCPU, 32 GiB). You can check this by executing:
+
+```
+kubectl get nodes --label-columns node.kubernetes.io/instance-type
+``` 
+After two minutes, the first job finishes and the pod is terminated:
+```
+$> kubectl get jobs
+NAME        COMPLETIONS   DURATION   AGE
+2-min-job   1/1           2m39s      2m40s
+5-min-job   0/1           2m40s      2m40s
+```
+```
+> $kubectl get pods
+NAME                   READY   STATUS    RESTARTS   AGE
+5-min-job-6ffsg        1/1     Running   0          2m50s
+nginx-8467c776-r8j24   1/1     Running   0          2m50s
+```
+Now, the total number of vCPU required by the running pods are **4 vCPU**:
+- NGINX server - 2 vCPU required
+-  5-minutes job - 2 vCPU required 
+
+The default behaviour is the one defined in the NodePool: `consolidationPolicy: WhenUnderutilized`. Karpenter identifies the **c6g.4xlarge** (12 vCPU) is underutilized, and perform a consolidation replacement of the node. It launches a cheaper and smaller node: a **c6g.2xlarge** (8 vCPU) instance. You can check these logs by executing the following command in another terminal:
+```
+kubectl -n karpenter logs -l app.kubernetes.io/name=karpenter --all-containers=true -f --tail=20
+```
+You should see these logs:
+```
+{"level":"INFO","time":"2024-01-10T15:06:37.063Z","logger":"controller.disruption","message":"disrupting via consolidation replace, terminating 1 candidates ip-10-0-93-19.eu-west-1.compute.internal/c6g.4xlarge/on-demand and replacing with on-demand node from types r6gd.2xlarge, c7i.2xlarge, r5a.2xlarge, m5a.2xlarge, m6a.2xlarge and 37 other(s)","commit":"1072d3b"}
+...
+{"level":"INFO","time":"2024-01-10T15:06:39.390Z","logger":"controller.nodeclaim.lifecycle","message":"launched nodeclaim","commit":"1072d3b","nodeclaim":"default-98tsh","nodepool":"default","provider-id":"aws:///eu-west-1a/i-0f329cada644371ec","instance-type":"c6g.2xlarge","zone":"eu-west-1a","capacity-type":"on-demand","allocatable":{"cpu":"7810m","ephemeral-storage":"17Gi","memory":"14003Mi","pods":"58","vpc.amazonaws.com/pod-eni":"38"}}
+```
+The NGINX server and the 5-min job pods are rescheduled into the new c6g.2xlarge node, so **the job is restarted**, which is not ideal for batch jobs.
```

Comment:
"which is not ideal for batch jobs" to "which will cause a disruption the job might not be prepared to handle like doing a checkpoint"